tplinkrouterc6u/dataclass.py

```diff
@@ -30,4 +30,10 @@ class Status:
     guest_5g_enable: bool
     wifi_2g_enable: bool
     wifi_5g_enable: bool
+    wan_ipv4_uptime: int
+    mem_usage: float
```

Comment:
could you add new fields `mem_usage` and `cpu_usage` to [ README.md ](https://github.com/AlexandrErohin/TP-Link-Archer-C6U/blob/main/README.md?plain=1#L76)